build.xml

```diff
@@ -351,6 +351,7 @@ Authors:
                    org/w3c/dom/html/HTMLDOMImplementation.class"
          manifest="${build.dir}/manifest.xerces">
       <manifest>
+        <attribute name="Automatic-Module-Name" value="org.apache.xerces.impl"/>
```

Comment:
Why not org.apache.xerces?

Reply:
I went with that because the jar is named xercesImpl.jar